### PR TITLE
fix(#3866): toast text color

### LINKED ISSUE #3866: VaToast incorrect text color
**Vuestic-ui version:** 1.8.0

Should probably be white for both text and icon. Please also check other examples.

image

### PR DESCRIPTION
**Pull request by m0ksem**
close #3866

### CODE REVIEW

File: packages/ui/src/components/va-toast/VaToast.stories.ts [hunk 1]
```diff
@@ -1,12 +1,28 @@
...
-  components: { VaToast },
-  template: '<VaToast/>',
+export const Default = () => defineComponent({
+  components: { VaToastDemo },
+  template: '<VaToastDemo/>',
 })
+
+export const TextColor: StoryFn = () => ({
+  components: { VaToast: VaToast },
+  template: '<VaToast color="red"/>',
```
Comment (m0ksem) [inline packages/ui/src/components/va-toast/VaToast.stories.ts:19]: @asvae

What should I do to test `<VaToast color="black" />` and `<VaToast color="white" />`?

Comment (asvae) [reply packages/ui/src/components/va-toast/VaToast.stories.ts:19]:
```
Color

[black]
VaToast color=black

[white]
VaToast color=white
```

Do we need this test though? I though it used general logic.

If yes you might also want to have no color as an option, i.e.

[primary?]
VaToast

[black]
VaToast color=black

[white]
VaToast color=white

Comment (m0ksem) [reply packages/ui/src/components/va-toast/VaToast.stories.ts]: Fixed.

**Review by asvae — APPROVED** [review]
I tweaked story a bit.

**Pull request merged**